release/create_release_archives.sh

```diff
@@ -34,6 +34,16 @@ COMMON_FILES_DIR="${TMP_DIR}/istio/istio-${ISTIO_VERSION}"
 ARCHIVES_DIR="${TMP_DIR}/archives"
 mkdir -p "${COMMON_FILES_DIR}/bin" "${ARCHIVES_DIR}"
 
+# On mac, brew install gnu-tar gnu-cp
+# and set CP=gcp TAR=gtar
+
+if [[ -z "${CP}" ]] ; then
+  CP=cp
+fi
+if [[ -z "${TAR}" ]] ; then
+  TAR=gtar
```

Comment:
you are right ! problem with only testing on my box :-) (though I think it's frequent to also have gtar on linux)